File: .ci/doc/templates/without-connect.tpl.java
try {
  AbstractProtocol protocol;
  if (System.getenv("SNIPPET_PROTOCOL") != null && System.getenv("SNIPPET_PROTOCOL").equals("http")) {
    protocol = new Http("kuzzle");
  } else {
    protocol = new WebSocket("kuzzle");
  }
  kuzzle = new Kuzzle(protocol);
  [snippet-code]
  System.out.println("Success");
} catch (Exception e) {
  System.err.println(e.getMessage());
} finally {
  if (kuzzle != null) {
    kuzzle.disconnect();
  }
}

File: .ci/doc/templates/without-connect.tpl.kt
val protocol: AbstractProtocol;
if (System.getenv("SNIPPET_PROTOCOL") != null && System.getenv("SNIPPET_PROTOCOL") == "http") {
  protocol = Http("kuzzle")
} else {
  protocol = WebSocket("kuzzle")
}
val kuzzle = Kuzzle(protocol).apply {
  connect()
}
try {
  [snippet-code]
  println("Success")
} catch (e: Exception) {
  e.printStackTrace()
} finally {
  kuzzle.disconnect()
}